### PR TITLE
[stable/openebs]: update openebs charts to 2.7.0

### PR DESCRIPTION
**Pull request by akhilerm**
Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

Fixes: https://github.com/openebs/openebs/issues/3338

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/openebs]`)

### CODE REVIEW

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by prateekpandey14 — APPROVED** [review]
lgtm

File: charts/openebs/values.yaml [hunk 1]
```diff
@@ -263,9 +263,17 @@ policies:
...
   # Specify in hours the duration after which a ping event needs to be sent.
   pingInterval: "24h"
+
+
+# To cleanup the validatingwebhook just before deletion of the charts
+# via a pre-delete webhook job
+hyperkubeImage:
+  repository: "k8s.gcr.io/hyperkube"
+  tag: "v1.12.1"
+  pullPolicy: "IfNotPresent"
```
Comment (kmova) [inline charts/openebs/values.yaml:279]: q_: please check if the tolerations can be added to the job?

Comment (akhilerm) [reply charts/openebs/values.yaml:279]: @prateekpandey14 @shubham14bajpai Can you check this. https://github.com/openebs/maya/blob/master/pkg/install/v1alpha1/jiva_volume.go#L1363

Comment (kmova) [reply charts/openebs/values.yaml:279]: We should probably reuse the tolertions set on m-apiserver instead of having users repeat the same thing again.

Comment (prateekpandey14) [reply charts/openebs/values.yaml:279]:
```yaml
{{- if .Values.webhook.tolerations }}
      tolerations:
{{ toYaml .Values.webhook.tolerations | indent 8 }}
{{- end }}


```

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Review by kmova — APPROVED** [review]

**Pull request merged**